src/components/Footer.jsx

```diff
@@ -20,15 +20,17 @@ export default function Footer() {
                 
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <h3>Made By:</h3><br />

```

Comment:
I removed full names for security purposes.

Reply:
Cool!